package-lock.json

```diff
@@ -12709,9 +12709,9 @@
       }
     },
     "nth-check": {
-      "version": "2.0.1",
-      "resolved": "https://registry.npmjs.org/nth-check/-/nth-check-2.0.1.tgz",
-      "integrity": "sha512-it1vE95zF6dTT9lBsYbxvqh0Soy4SPowchj0UBGj/V6cTPnXXtQOPUbhZ6CmGzAD/rW22LQK6E96pcdJXk4A4w==",
+      "version": "2.0.0",
```

Comment:
Looked like it reverted some of your latest landed PRs

Reply:
Odd, will take another look at that

Reply:
I think you just need to rebase off the last commit

Reply:
This should be good now. It seems running `npm install` currently will revert some references of `nth-check` and `trim-newlines`. An `npm audit fix` should correct this

Reply:
Opened PR #2158 for that @nschonni